nBTC/sources/redeem_request.move

```diff
@@ -136,6 +142,19 @@ public(package) fun move_to_signing_status(r: &mut RedeemRequest, redeem_id: u64
     });
 }
 
+public(package) fun move_to_confirmed_status(
+    r: &mut RedeemRequest,
+    redeem_id: u64,
+    tx_id: vector<u8>,
+) {
+    r.status = RedeemStatus::Confirmed;
+    //TODO: Review what to emit for this event
+    event::emit(ConfirmedEvent {
+        id: redeem_id,
+        tx_id: tx_id,
+    });
```

Comment:
why not having a more generic event:

```
RedeemReqStatusEvent {
  id,
  status,
}
```

NOTE: we don't need to put tx_id - events are linked to the transactions anyway